operators/datadog-operator-certified/1.3.0/manifests/datadog-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1024 @@
+                    name: manager
+                    ports:
+                      - containerPort: 8080
+                        name: metrics
+                        protocol: TCP
+                    resources:
+                      limits:
+                        cpu: 100m
+                        memory: 250Mi
+                      requests:
+                        cpu: 100m
+                        memory: 250Mi
+                serviceAccountName: datadog-operator-controller-manager
+                terminationGracePeriodSeconds: 10
+      permissions:
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps/status
+              verbs:
+                - get
+                - update
+                - patch
+            - apiGroups:
+                - ""
+              resources:
+                - events
+              verbs:
+                - create
+                - patch
+            - apiGroups:
+                - coordination.k8s.io
+              resources:
+                - leases
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - coordination.k8s.io
+              resources:
+                - leases/status
+              verbs:
+                - get
+                - update
+                - patch
+          serviceAccountName: datadog-operator-controller-manager
+    strategy: deployment
+  installModes:
+    - supported: false
+      type: OwnNamespace
+    - supported: false
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - Datadog
+    - Monitoring
+    - Logs
+    - Tracing
+  links:
+    - name: Documentation
+      url: https://docs.datadoghq.com/agent/kubernetes/
+    - name: Kubernetes Monitoring Info
+      url: https://www.datadoghq.com/blog/tag/kubernetes/
+  maintainers:
+    - email: support@datadoghq.com
+      name: Datadog Inc.
+  maturity: alpha
+  minKubeVersion: 1.16.0
+  provider:
+    name: Datadog
+  relatedImages:
+    - image: registry.connect.redhat.com/datadog/operator@sha256:84840ae4daf904c9620e93dff0e443d884259ffd35f200c864eadaabb18a28c0
+      name: manager
+    - image: registry.connect.redhat.com/datadog/operator@sha256:84840ae4daf904c9620e93dff0e443d884259ffd35f200c864eadaabb18a28c0
+      name: operator-84840ae4daf904c9620e93dff0e443d884259ffd35f200c864eadaabb18a28c0-annotation
+  replaces: 1.2.1
```

Comment:
This line needs to be changed to `replaces: datadog-operator.v1.2.1`

Reply:
Thanks for a quick feedback! Pushed the fix.